frontend/build/shared/constants.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-env node */
+
+const WEBPACK_BUNDLE = process.env.WEBPACK_BUNDLE || "development";
```

Comment:
tiny nit: probably good to export this so we only have the `|| development` fallback defined in one place